Seq.Client.Portable/Client/Portable/PortableSeqSink.cs

```diff
@@ -35,17 +36,18 @@ class PortableSeqSink : PortablePeriodicBatchingSink
         public const int DefaultBatchPostingLimit = 1000;
         public static readonly TimeSpan DefaultPeriod = TimeSpan.FromSeconds(2);
 
-        public PortableSeqSink(string serverUrl, string apiKey, int batchPostingLimit, TimeSpan period)
+        public PortableSeqSink(string serverUrl, string apiKey, int batchPostingLimit, TimeSpan period, IHttpClientFactory httpClientFactory)
             : base(batchPostingLimit, period)
         {
             if (serverUrl == null) throw new ArgumentNullException("serverUrl");
             _apiKey = apiKey;
+            _httpClientFactory = httpClientFactory;
 
             var baseUri = serverUrl;
             if (!baseUri.EndsWith("/"))
                 baseUri += "/";
 
-            _httpClient = new HttpClient { BaseAddress = new Uri(baseUri) };
+            _httpClient = _httpClientFactory.CreateHttpClient(new Uri(baseUri));
```

Comment:
It was mainly just in case someone wanted to configure the httpclient differently based on the url.. for instance bypass the proxy for particular urls. Not certain that its really needed though it was just an after thought so happy to remove if you think best.